js/locales/udata.fr.json

```diff
@@ -307,6 +307,8 @@
     "Role": "Rôle",
     "Role \"{role}\" is required": "Le rôle \"{role}\" est requis",
     "Roles": "Rôles",
+    "Run": "Run",
```

Comment:
I think we're missing some trads on crowdin: https://crowdin.com/project/udata/fr